mindmeld/ser.py

```diff
@@ -216,16 +229,17 @@ def parse_numerics(
             # Convert a second grain unix timestamp to millisecond
             timestamp *= 1000
         data["reftime"] = timestamp
-    return SystemEntityRecognizer.get_instance().get_response(data)
+    return DucklingRecognizer.get_instance(url=url).get_response(data)
```

Comment:
Can we do this in a way that doesn't assume the entity recognizer is Duckling? Like if someone has their own special system entity recognizer for Japanese, how would they use this function? Perhaps instead of passing in `url` we could pass in a SystemEntityRecognizer, defaulting to Duckling.

Reply:
I guess we could, but `parse_numerics` itself (and its immediate callers) is highly specific to Duckling. So one way to do it is to look at the caller of functions such as `get_candidates` and `resolve_system_entity` and see if it makes sense to allow a SystemEntityRecognizer to be passed in there, and if none is passed we need to default to Duckling (with the default URL).

Maybe in a follow-up PR we can explore an actual entity recognizer that is not Duckling (such as spacy). We should note that Duckling is currently doing **both** entity recognition and entity resolution, so whatever alternative that we pick must satisfy both.